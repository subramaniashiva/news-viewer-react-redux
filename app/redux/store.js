/*
  Redux store file.
  Combines the root reducer, thunk middleware (for API requests)
*/
import { createStore } from 'redux';

import rootReducer from './reducers/rootReducer';

export default createStore(rootReducer, {});
