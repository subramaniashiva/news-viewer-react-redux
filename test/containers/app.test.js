import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import App from '../../app/containers/app';
import store from '../../app/redux/store';

describe('Main container', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <App />
      </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});