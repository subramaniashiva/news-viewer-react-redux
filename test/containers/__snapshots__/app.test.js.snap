// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main container renders correctly 1`] = `
<div>
  <div>
    React app
  </div>
</div>
`;
